benchmarks/helics/PholdFederate.hpp

```diff
@@ -39,60 +34,58 @@ class PholdFederate {
     double lookahead_ = deltaTime * .1;
 
     // classes related to the exponential and uniform distribution random number generator
-    bool generateRandomSeed = true;
+    bool generateRandomSeed = false;
     unsigned int seed =
         0xABad5eed; // some suggestions for seed choice were that not having a majority of the bits as 0 is better
     std::mt19937 rand_gen;
     std::exponential_distribution<double> rand_exp;
     std::uniform_real_distribution<double> rand_uniform_double;
     std::uniform_int_distribution<unsigned int> rand_uniform_int;
 
-    // callbacks for instrumenting the code
-    std::function<void()> callBeforeFinalize = nullptr;
-    std::function<void()> callAfterFinalize = nullptr;
-
-    bool initialized{false};
-    bool readyToRun{false};
-
   public:
-    PholdFederate() = default;
+    // TODO: output cluster name in a summary file, along with node count and feds per node
```

Comment:
does this TODO need to be done in this PR or is that something that will wait a while?

Reply:
This is something that a job launcher PR will have.